src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todoInfo: Todo | null;
+  setTodoInfo: React.Dispatch<React.SetStateAction<Todo | null>>;
+};
+
+export const TodoModal: React.FC<Props> = ({ todoInfo, setTodoInfo }) => {
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [loading, setLoading] = useState(true);
```
to fix that blinking when opening the modal for the first time